torch/testing/_internal/common_methods_invocations.py

```diff
@@ -755,7 +755,7 @@ def np_unary_ufunc_integer_promotion_wrapper(fn):
 
     # Helper to determine if promotion is needed
     def is_integral(dtype):
-        return dtype in [np.bool, np.uint8, np.int8, np.int16, np.int32, np.int64]
+        return dtype in [bool, np.uint8, np.int8, np.int16, np.int32, np.int64]
```

Comment:
Note that there is a numpy-native boolean dtype:
```
>>> np.bool is bool
True
>>> np.bool_
<class 'numpy.bool_'>
```

Reply:
Thanks, I also added `np.bool_` here.